src/facts/server/authorization_sql_builder.ts

```
@@ -62,10 +64,10 @@ export default class AuthorizationSqlBuilder {
         object as node
         FROM facts
         WHERE predicate IN (${groupRoles.join(',')})
-        AND fact_box_id IN (${factScope.factBoxIds.join(',')})
+        AND fact_box_id IN (${factScope.factBoxIds.map(EnsureIsValid.factBoxId).join(',')})
         AND subject IN (${await this.getGroupsOfTheUser(userid, membershipType, factScope)})`;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)